packages/create-contentful-app/template/bin/prompt-app-definition.js

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+
+const inquirer = require('inquirer');
+const path = require('path');
+const createAppDefinition = require('./create-app-definition');
+
+let appDefinitionSettings;
+
+(async () => {
+  appDefinitionSettings = await inquirer.prompt([
+    {
+      name: 'name',
+      message: `App name (${path.basename(process.cwd())}):`
+    },
+    {
+      name: 'locations',
+      message: 'Select locations for this app:',
+      type: 'checkbox',
+      choices: [
```

Comment:
How about including app config screen location? You can now have an app with or without the screen

Reply:
It's more that app-config will be auto-included as these are frontend apps. It also is a cheap way for them to see the app for the first time working. But we left a comment in index.tsx explaining they can remove it later.